dask_expr/_groupby.py

```diff
@@ -1095,6 +1096,7 @@ def _lower(self):
             True,
             False,
             True,
+            None,
```

Comment:
Oh man, what's the policy on named kwargs? Looking at a list of (True, True, False, True, None, ..) is not fantastic.

Reply:
100% agree

There is no real policy, its a technical necessity as long as we have the dynamic thing without constructors

If you need *args like here a couple lines below, you can't pass kwargs because the constructor can only deal with it like this